src/components/Graph.tsx

```diff
@@ -17,6 +17,11 @@ interface GraphProps {
    * callback function when click node
    */
   readonly focusedNodeID?: string;
+
+  /**
+   * callback function when mouse hover node
```

Comment:
Let's improve this comment, it is called when mouse hover our and out on a node data.